apps/explorer/components/SCToFiatDisplay.tsx

```diff
@@ -0,0 +1,33 @@
+'use client'
+
+import {
+  currencyOptions,
+  useAppSettings,
+  useSiascanExchangeRate,
+} from '@siafoundation/react-core'
+import { siacoinToFiat } from '../lib/currency'
+import { Skeleton } from '@siafoundation/design-system'
+
+type Props = {
+  sc: string
+}
+
+export function SCToFiatDisplay({ sc }: Props) {
+  const {
+    settings: { currency },
+  } = useAppSettings()
+
+  const { rate, error, isLoading } = useSiascanExchangeRate({
+    currency: currency.id,
+    api: 'https://api.beta.siascan.com/zen/api',
```

Comment:
Also maybe just  remove `zen`, not sure the difference but might as well use `mainnet` exchange rates.